config/ironic.conf.j2

```diff
@@ -1,20 +1,19 @@
 [DEFAULT]
 auth_strategy = noauth
 debug = true
-default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_bios_interfaces = idrac-wsman,no-bios,redfish,idrac-redfish,irmc
-enabled_boot_interfaces = pxe,ipxe,fake,redfish-virtual-media,idrac-redfish-virtual-media
+enabled_bios_interfaces = idrac-wsman,no-bios,redfish,idrac-redfish,irmc,ilo
+enabled_boot_interfaces = ipxe,ilo-ipxe,pxe,ilo-pxe,fake,redfish-virtual-media,idrac-redfish-virtual-media,ilo-virtual-media,ilo-uefi-https
```

Comment:
Let's not add `ilo-uefi-https`, we have no code to support it in BMO.

Reply:
I've removed `ilo-uefi-https`.